libs/site-base/internal/webpack/webpack.prod.babel.js

```diff
@@ -98,6 +98,9 @@ const prodOptions = {
     assetFilter: assetFilename =>
       !/(\.map$)|(^(main\.|favicon\.))/.test(assetFilename),
   },
+  experiments: {
+    topLevelAwait: true,
+  }
 };
```

Comment:
Isn't currently used on prod (only TLA is in dev:live specific code) but better to be consistent